Assets/MRTK/SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -63,6 +63,22 @@ public LayoutAnchor Anchor
             set { anchor = value; }
         }
 
+        [SerializeField, Tooltip("How the columns are aligned in the grid")]
+        private LayoutHorizontalAlignment colAlignment = LayoutHorizontalAlignment.Center;
+        public LayoutHorizontalAlignment ColAlignment 
```

Comment:
Would be good to add `<summary>`s to the public properties. The text can be the same as the corresponding field's tooltip